src/shared/components/oncoprint/DeltaUtils.ts

```diff
@@ -448,7 +448,7 @@ function transitionClinicalTrack(
             data: nextSpec.data,
             data_id_key: "uid",
             label: nextSpec.label,
-            description: nextSpec.description,
+            description: (nextSpec.label.trim() === nextSpec.description.trim()) ? "" : nextSpec.description,
```

Comment:
are we 100% sure these props are both non-null?

Reply:
good point

Reply:
yes we are :)

Reply:
the reason i ask is that we know client generator isn't accurate in reflecting the true "nullability" of fields and "description" strikes me as something that we might know is sometimes missing.  would be worth checking

Reply:
btw, i'm assuming description comes from api data though i can't see it.

Reply:
good catch!

Reply:
okay I updated it